pkg/relayer/session/sessiontree_test.go

```diff
@@ -68,29 +79,19 @@ func TestSessionTree_CompactProofsAreSmallerThanNonCompactProofs(t *testing.T) {
 			err = gzipWriter.Close()
 			require.NoError(t, err)
 
-			// Accumulate the gzipped proof size over 1000 runs.
+			// Accumulate the gzipped proof size over numIterations runs.
 			cumulativeGzippedProofSize += len(buf.Bytes())
-
-			//t.Logf(
-			//	"numLeaf: %d, proofSize: %d, compactProofSize: %d, gzipProofSize: %d",
-			//	numLeaf, len(proofBz), len(compactProofBz), len(buf.Bytes()),
-			//)
-
-			// Commenting out the assertion to not fail the test since compaction is not
-			// guaranteed to always reduce the proof size.
-			//require.Less(t, len(compactProofBz), len(proofBz))
 		}
 
-		//t.Logf(
-		//	"numLeaf=%d: cumulativeProofSize: %d, cumulativeCompactProofSize: %d, cumulativeGzippedProofSize: %d",
-		//	numLeaf, cumulativeProofSize, cumulativeCompactProofSize, cumulativeGzippedProofSize,
-		//)
+		compactionRatio := float32(cumulativeCompactProofSize) / float32(cumulativeProofSize)
+		compressionRatio := float32(cumulativeGzippedProofSize) / float32(cumulativeProofSize)
+
+		// Gzip compression is more efficient than compaction.
+		require.Greater(t, compactionRatio, compressionRatio)
```

Comment:
This seems to contradict the comment above, no?

Reply:
Inverted how the ratio is calculated, now the higher the better.